test/extended/fixtures/forcepull-false-cust.json

```diff
@@ -0,0 +1,384 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "ruby-helloworld-sample",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
```

Comment:
there's not much overlap between services, deploymentconfigs, and buildconfigs...  so i'm having a hard time thinking of anything we'd be likely to turn up by creating those additional objects.

using imagestreams (as opposed to direct docker references) does make sense from a coverage perspective (but in that case i'd agree that you probably want to do a mixture)